### PR TITLE
fix: catch lockfile parse errors

### PR DESCRIPTION
**Pull request by mrackwitz**
- [x] Tests written and linted [ℹ︎](https://github.com/snyk/general/wiki/Tests)
- [x] ~~Documentation written [ℹ︎](https://github.com/snyk/general/wiki/Documentation)~~
- [x] Commit history is tidy [ℹ︎](https://github.com/snyk/general/wiki/Git)
- [x] https://github.com/snyk/cocoapods-lockfile-parser/pull/14 was merged and shipped

### What this does

This catches when a lockfile is invalid and throws a nice error, explaining the user what actually happened, because otherwise the error message will have no context.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   "homepage": "https://github.com/snyk/snyk-cocoapods-plugin#readme",
   "dependencies": {
     "@snyk/cli-interface": "^1.5.0",
-    "@snyk/cocoapods-lockfile-parser": "^2.0.0",
+    "@snyk/cocoapods-lockfile-parser": "^2.0.2",
```
Comment (darscan) [inline package.json]: I would make all 3 of these exact (no ranges), so that we can control our releases

Comment (darscan) [reply package.json]: by all 3 I mean:
```
@snyk/cli-interface
@snyk/cocoapods-lockfile-parser
@snyk/dep-graph (which should be 1.13.0)
```

Comment (mrackwitz) [reply package.json]: I guess, given how the snyk CLI is distributed, that does make sense.

Comment (mrackwitz) [reply package.json]: Really dep-graph 1.13.0? Snyk still ships with 1.12.0?

Comment (darscan) [reply package.json]: It just adds a new method

**Pull request merged**